### ISSUE TITLE
Pretrained checkpoints

### ISSUE THREAD

**Issue by wonozlo** [junsang7777/VOS-VFI#2]
Hi,

thanks for your nice work and well organized codes!

I want to test my own dataset with your model, but I cannot find checkpoint for test.

Can you provide checkpoint of adacof-vos for my toy test?

Thanks.

**Comment by junsang7777** [comment]
" Download STCN pretrained model in saves and AdaCoF-VOS our pretrain model." Would you like to click the AdaCoF-VOS part here?

**Comment by wonozlo** [comment]
Oh I missed it. Thanks! And also, do I need to run train code to get the corresponding config file?

**Comment by junsang7777** [comment]
Currently, it has the same settings as the adacof base model. Therefore, no config file is required. But, you can change the hyperparameter of adacof or our loss. (ex. change segmentation loss weights in Trainer.py 118.  seg_loss = c*seg_forward_loss + a*seg_backward_loss + b*seg_consistent_loss)
